plugins/inputs/nsq_consumer/nsq_consumer.go

```diff
@@ -134,15 +135,28 @@ func (n *NSQConsumer) Start(ac telegraf.Accumulator) error {
 		return nil
 	}))
 
+	// For backward compatibility
+	if n.Server != "" {
+		n.Nsqd = append(n.Nsqd, n.Server)
```

Comment:
documentation says we _prepend_ it to the list, but we're appending it here. Could be a concern if the server connection list order matters?

Reply:
First of all thanks a lot for the review!
Regarding the order, I just used what was done before in this plugin (previous line 143). I can change this, should I?

Reply:
We should probably change the docs to say append. If the plugin appended before, changing it to prepending might cause incompatibility.